pages/StartView.page.ts

```diff
@@ -1,17 +1,35 @@
-import { type Locator, Page } from '@playwright/test';
+import { type Locator, Page } from "@playwright/test";
+import { MenuPage } from "./MenuBar.page";
+import { LandingPage } from "./LandingView.page";
 
-export class StartPage{
-    readonly page: Page;
-    readonly text_h2_Ueberschrift: Locator;
-    readonly card_item_email: Locator;
-    readonly card_item_itslearning: Locator;
-    readonly card_item_schulportal_administration: Locator;
+export class StartPage {
+  readonly page: Page;
+  readonly text_h2_Ueberschrift: Locator;
+  readonly card_item_email: Locator;
+  readonly card_item_itslearning: Locator;
+  readonly card_item_schulportal_administration: Locator;
 
-    constructor(page){
-        this.page = page;  
-        this.text_h2_Ueberschrift = page.getByTestId('all-service-provider-title');
-        this.card_item_email = page.locator('[href="https://de.wikipedia.org/wiki/E-Mail"]').first();
-        this.card_item_itslearning = page.locator('[href="https://sh-staging.itslintegrations.com/"]');
-        this.card_item_schulportal_administration = page.getByText('Schulportal-Administration');
-    }
-}
\ No newline at end of file
+  constructor(page) {
+    this.page = page;
+    this.text_h2_Ueberschrift = page.getByTestId("all-service-provider-title");
+    this.card_item_email = page
+      .locator('[href="https://de.wikipedia.org/wiki/E-Mail"]')
+      .first();
+    this.card_item_itslearning = page.locator(
+      '[href="https://sh-staging.itslintegrations.com/"]',
+    );
+    this.card_item_schulportal_administration = page.getByText(
+      "Schulportal-Administration",
+    );
+  }
+
+  public async administration(): Promise<MenuPage> {
+    await this.card_item_schulportal_administration.click();
+    return new MenuPage(this.page);
+  }
+
+  public async start(): Promise<LandingPage> {
+    await this.page.goto(process.env.FRONTEND_URL || "");
```

Comment:
I don't know how playwright handles the goto, but I feel like the fallback value should be `"/"` to make sure the location is root